backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/material/cockpit/filters/MaterialCockpitFilters.java

```diff
@@ -142,10 +142,11 @@ private boolean augmentQueryBuilder(final IQueryBuilder<I_MD_Cockpit> queryBuild
 	private IQueryBuilder<I_MD_Cockpit> augmentQueryBuilderWithOrderBy(@NonNull final IQueryBuilder<I_MD_Cockpit> queryBuilder)
 	{
 		return queryBuilder
+				.orderByDescending(I_MD_Cockpit.COLUMNNAME_QtyStockEstimateSeqNo_AtDate)
 				.orderBy()
-				.addColumn(I_MD_Cockpit.COLUMN_DateGeneral)
+				.addColumn(I_MD_Cockpit.COLUMNNAME_DateGeneral)
```

Comment:
why not simply use the IQueryBuilder.orderBy(columnname) method?
similar to IQueryBuilder.orderByDescending(columnname).
so you don't need to call "orderBy()".

Reply:
thx, done